templates/html/templates/common/macros.html

```diff
@@ -60,7 +60,7 @@
     {% endif %}
 {%- endmacro %}
 
-{% macro render_description_item_plain_literal(result_set, property_id, property_label, accordion_style=False) -%}
+{% macro render_description_item_plain_literal(result_set, property_id, property_label, rdfa_property, accordion_style=False, code_mode=False, accordion_content_state="hide") -%}
```

Comment:
Please turn this parameter into a boolean one. To make it readable rename the parameter into `is_accordeon_expaded` taking true/false values.